File: algorithm/src/main/java/baekjoon/p10103/Main.java
package baekjoon.p10103;

import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();

        int[][] inputs = new int[n][2];
        for (int i = 0; i < inputs.length; i++) {
            inputs[i][0] = sc.nextInt();
            inputs[i][1] = sc.nextInt();
        }

        for (int num : solve(inputs)) {
            System.out.println(num);
        }
    }

    private static int[] solve(int[][] inputs) {
        int[] ret = new int[]{100, 100};

        for (int[] input : inputs) {
            if (input[0] < input[1]) {
                ret[0] -= input[1];
            } else if (input[0] > input[1]) {
                ret[1] -= input[0];
            }
        }

        return ret;
    }

}

File: algorithm/src/main/java/baekjoon/p10103/Main.kt
package baekjoon.p10103

fun main(args: Array<String>) {
    val n = readLine()!!.toInt()
    val inputs = Array(n) { IntArray(2) }

    for (input in inputs) {
        val (a, b) = readLine()!!.split(" ").map { it.toInt() }
        input[0] = a
        input[1] = b
    }

    for (ret in solve(inputs)) {
        println(ret)
    }
}

private fun solve(inputs: Array<IntArray>): IntArray {
    val ret = IntArray(2) { 100 }

    for (input in inputs) {
        if (input[0] < input[1]) {
            ret[0] -= input[1]
        } else if (input[0] > input[1]) {
            ret[1] -= input[0]
        }
    }

    return ret
}